sandy/errorr.py

```diff
@@ -183,8 +197,9 @@ def get_cov(self):
                     jvals = ix.query("MAT==@mat & MT==@mt1").squeeze()
                     jmin, jmax = jvals.IMIN, jvals.IMAX
                     c[imin: imax, jmin: jmax] = cov
-                    c[jmin: jmax, imin: imax] = cov.T
-            
+                    if mt1 != mt:
```

Comment:
Is this needed?

Reply:
If the section of covariance matrix is already in diagonal position line 186 is useless, but it doesn't create issues so I can also leave it as it was.